lib/chat/ui/widgets/date_picker.dart

```diff
@@ -0,0 +1,241 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class _Constants {
+  static const defaultMainAxisAlignment = MainAxisAlignment.start;
+  static const defaultIsDenseList = true;
+  static const defaultListTilePadding = const EdgeInsets.all(0);
+  static const defaultLeftIconHeight = 20.0;
+  static const defaultSizedBoxSeparatorWidth = 22.0;
+  static const defaultPickerRowMainAxisAlignment =
+      MainAxisAlignment.spaceBetween;
+  static const defaultTrailingImageHeight = 13.0;
+  static const defaultPickerDescriptionStyle = const TextStyle(
+    fontSize: 17,
+    fontWeight: FontWeight.w400,
+    color: Color(0xFF1A1B46),
+  );
+  static const defaultPickerValueStyle = const TextStyle(
+    fontSize: 15,
+    fontWeight: FontWeight.normal,
+    color: Color(0xFF767690),
+  );
+
+  static final currentDate = DateTime.now();
+  static final minDateLimit = DateTime(2019, 6);
+  static final maxDateLimit = DateTime(2101);
+  static final dateFormatter = DateFormat('dd MMMM yyyy');
```

Comment:
Not sure that we have to use this format, I think that we should use `MM/DD/YYYY`, is how it is in the Form used to fill the JSON data:

<image src="https://user-images.githubusercontent.com/15966505/62873704-2e9aa280-bd20-11e9-9c9e-9036cfba1dc1.png" width=60%/>

Reply:
As an observation, I've used the already existing format from code base what was defined in date picker.